README.md

```diff
@@ -73,24 +56,221 @@ The `-v /path/to/your/local/models:/workspace/models` part will make sure
 the downloaded models are cached to your host storage. 
 which makes rerunning stuff much faster.
 
-The `-v ${PWD}:/workspace/spacer/` mounts your current folder including 
-`secrets.json` so that the container has the right permissions.
+The `-v ${PWD}:/workspace/spacer/` mounts your current folder (including 
+`secrets.json`, if used) so that the container has the right permissions.
 
 The last step will run the default CMD command specified in the dockerfile 
-(unit-test with coverage). If you want to enter the docker container 
+(unit-test with coverage). If you want to enter the docker container, 
 run the same command but append `bash` in the end.
 
-#### Pip install
+### Pip install
 * `pip install pyspacer`
-* Set environmental variables.
+* Set up configuration. `secrets.json` isn't an option here, so either use environment variables, or Django settings if you have a Django project.
 
-#### Local clone
+### Local clone
 * Clone this repo
+  * If using Windows: turn Git's `autocrlf` setting off before your initial checkout. Otherwise, pickled classifiers in `spacer/tests/fixtures` will get checked out with `\r\n` newlines, and the pickle module will fail to load them, leading to test failures. However, autocrlf should be left on when adding any new non-pickle files.
```

Comment:
This is extremely helpful (though none of us use windows...)

Reply:
Now that I think of it, these pickled classifiers might be better moved to Git LFS, in which case Git might not try to do anything with newlines at all. Hopefully! (But Git LFS is for another PR.)